src/canary_server.cpp

```diff
@@ -342,8 +345,12 @@ void CanaryServer::loadModules() {
 	}
 
 	auto coreFolder = g_configManager().getString(CORE_DIRECTORY, __FUNCTION__);
-	// Load appearances.dat first
+// Load items dependencies
+#if CLIENT_VERSION < 1100
+	modulesLoadHelper(Item::items.loadFromOtb("data/items/items.otb"), "items.otb");
+#else
```

Comment:
This condition needs to be reviewed. You made the changes to use only the appearances.dat file and this will cause an error during server startup.

```suggestion
// Load items dependencies
#if CLIENT_VERSION < 1100
	modulesLoadHelper((g_game().loadAppearanceProtobuf(coreFolder + "/items/appearances.dat") == ERROR_NONE), "appearances.dat");
#endif
```